mars/core/operand/fuse.py

```diff
@@ -25,6 +25,11 @@ class Fuse(Operand):
 
     fuse_graph = ReferenceField("fuse_graph", ChunkGraph)
 
+    def _get_logic_key_token_values(self):
```

Comment:
Now the fuse op will no longer be generated during fusing when coloring algorithm is introduced.

Reply:
So we should remove the fuse op?